graphql/resolvers.go

```diff
@@ -63,6 +64,29 @@ func (r *hostResolver) Elapsed(ctx context.Context, obj *restModel.APIHost) (*ti
 	return obj.RunningTask.StartTime, nil
 }
 
+// func (r *hostResolver) InstanceTags(ctx context.Context, obj *restModel.APIHost) ([]*InstanceTag, error) {
+// 	return obj.InstanceTags, nil
+// }
+
+func (r *hostResolver) Distro(ctx context.Context, obj *restModel.APIHost) (*restModel.APIDistro, error) {
+
+	hostDistro, err := distro.FindByID(*obj.Distro.Id)
```

Comment:
is making this additional network request for host's distro necessary since the entire distro model is included in the host model? it can be accessed via api model [here](https://github.com/evergreen-ci/evergreen/blob/master/rest/model/host.go#L128)

Reply:
The distro thats included in the apiHost model only includes 3 fields, `id`, `Provider` and `imageId` I need info about the actual distro itself such as workDir and that needs to be queried for.

Reply:
`DistroInfo` only includes three fields. you can expand it to include whatever fields you need and then include them in `buildFromHostStruct`